chapters/annotations.tex

```diff
@@ -888,6 +888,13 @@ \subsection{Component Instance}\label{component-instance}
 Sub-connectors in a hierarchical connector are only shown when they can be connected to.
 Non-connector components are only shown in the diagram layer.
 
+\subsubsection{Default Outline for Missing Graphics}\label{default-outline-for-missing-graphics}
+If the icon of a component placed in a diagram layer does not contain any graphical primitives (including inherited ones, and regardless of visibility; but excluding connectors), tools shall show a tool-dependent rudimentary outline of the component's transformed \lstinline!extent!.
+
+\begin{nonnormative}
+The reason for making the tool-dependent outline rudimentary is to encourage the model developer to provide a proper icon.
+\end{nonnormative}
```

Comment:
```suggestion
\begin{nonnormative}
The reason for making the tool-dependent outline rudimentary is to encourage the model developer to provide a proper icon.
Regardless of visibility imply that if the primitives have parameter-dependent visibility it will not switch to this default icon regardless of the parameter setting.
\end{nonnormative}
```

Reply:
I think this gives more background regarding "regardless of visibility".

Reply:
Why exclude the simple constant case of `visible = false`?

Reply:
> Why exclude the simple constant case of `visible = false`?

I added that case as well.

Reply:
OK.  Then I think that it is only this question that remains (quoting myself):
> Could we also make this more concretely related to `visible`, or are there additional mechanisms that should also be considered?

Reply:
I was looking for a normative formulation in terms of `visible` if possible, or otherwise a normative explanation of what determining _visibility_ comes down to.

Reply:
Oh, I see. Is the new change clearer?

Reply:
I think we need to start with the normative part, so I initiated another conversation there.